lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt

```diff
@@ -100,7 +100,8 @@ internal open class AndroidExecutor(
             is Action.ShellBeforeAllScreens -> action.shell.shell()
             is Action.ShellBeforeScreen -> action.shell.shell()
             is Action.Swipe -> swipe(action.swipeMove)
-            Action.ResetGfxInfo -> "dumpsys gfxinfo $apkPackage --reset"
+            Action.ResetGfxInfo -> "dumpsys gfxinfo $apkPackage --reset".adbShell()
+            is Action.OpenAppAgain -> "monkey -p $apkPackage -c android.intent.category.LAUNCHER 1".adbShell()
```

Comment:
Maybe just OpenApp?

Reply:
not really, you open app with some activity before all actions. 
this command helps you to resume it after some pause in usage. currently it can be `openHomeScreen`